cli/flox-rust-sdk/src/models/environment/path_environment.rs

```diff
@@ -401,11 +424,73 @@ impl PathEnvironment {
             Err(EnvironmentError2::WriteEnvJson(e))?;
         }
 
-        // write "run" >> .flox/.gitignore
-        fs::write(dot_flox_path.join(".gitignore"), "run/\n")
-            .map_err(EnvironmentError2::WriteGitignore)?;
+        // write "run" and "cache" to .flox/.gitignore
+        fs::write(dot_flox_path.join(".gitignore"), formatdoc! {"
+            {GCROOTS_DIR_NAME}/
+            {CACHE_DIR_NAME}/
+            "})
+        .map_err(EnvironmentError2::WriteGitignore)?;
+
+        let mut environment = Self::open(pointer, dot_flox_path, temp_dir)?;
+
+        if let Some(InitCustomization {
+            packages: Some(packages),
+            ..
+        }) = customization
+        {
+            // Ignore the result, because we know there can't be packages already installed
+            environment.install(packages, flox)?;
+        }
+
+        Ok(environment)
+    }
+
+    /// Replace all placeholders in the manifest file contents
+    ///
+    /// TODO: we should probably be using toml_edit
```

Comment:
Yes, we should 😌

I have a small branch doing this with toml edit which looks rather concise.
When we know the input, we can use `["filed"]` accessors to much more easily address fields.

The main issue is, our comments are currently _prefix_ decorations of the _next_ item unfortunately.
That means we can't just remove the comments from the template (easily) and still have to add either fields or comments in here.
That all feels a bit stitched together from different places, but I guess it kinda has to be.

Reply:
https://github.com/flox/flox/issues/1064